pkg/Cpanel/Security/Advisor/Assessors/Apache.pm

```diff
@@ -68,6 +68,13 @@ sub _check_for_apache_chroot {
 
     my $security_advisor_obj = $self->{'security_advisor_obj'};
 
+    # See Cpanel::HttpUtils::ApRestart for where this regex is from
+    my $names_list = join( '|', map { '^' . $_, '/' . $_ } qw{litespeed lscgid} );
+    my $ls_regexp  = qr/$names_list/i;
+
+    require Cpanel::PsParser;
+    my $litespeed_running = Cpanel::PsParser::get_pids_by_name( $ls_regexp, [ "root", "nobody" ] );
```

Comment:
The way we check for litespeed elsewhere is to look for /usr/local/lsws/bin/lswsctrl and if it's present run it with the "status" argument to see if it's running. That's not a whole lot more reliable than the approach you're using here, but it still might be a good idea to maintain consistency with the rest of cPanel & WHM.